packages/fast-tooling-react/src/form/controls/control.button.tsx

```diff
@@ -52,6 +52,8 @@ class ButtonControl extends React.Component<
         e: React.MouseEvent<HTMLButtonElement>
     ) => void) => {
         return (e: React.MouseEvent<HTMLButtonElement>): void => {
```

Comment:
Not really this change - but does this really need to *return* a function?

Reply:
Good point, i'll take a look at it on my next pass